docs/documentation/recipes/using-docfx-and-tools.md

```diff
@@ -0,0 +1,245 @@
+```
+
+## 2.  Configure DocFx
+
+Configuration for DocFx is done in a `docfx.json` file. Store this file in the root of your repository.
+
+> **NOTE:** You can store the docfx.json somewhere else in the hierarchy, but then you need to provide the path of the file as an argument to the docfx command so it can be located.
+
+Below is a good configuration to start with, where documentation is in the **/docs** folder and the sources are in the **/src** folder:
+
+```json
+{
+    "metadata": [
+    {
+          "src": [
+          {
+              "files": [ "src/**.csproj" ],
+              "exclude": [ "_site/**", "**/bin/**", "**/obj/**", "**/[Tt]ests/**" ]
+          }
+          ],
+          "dest": "reference",
+          "disableGitFeatures": false
+       }
+    ],
+    "build": {
+        "content": [
+            { "files": [ "reference/**" ] },
+            {
+                "files": [ "**.md", "**/toc.yml" ],
+                "exclude": [ "_site/**", "**/bin/**", "**/obj/**", "**/[Tt]ests/**" ]
+            }
+        ],
+        "resource": [
+            { "files": ["docs/.attachments/**"] },
+            { "files": ["web.config"] }
+        ],
+        "template": [ "templates/cse" ],
+        "globalMetadata": {
+            "_appTitle": "CSE Documentation",
+            "_enableSearch": true
+        },
+        "markdownEngineName": "markdig",
+        "dest": "_site",
+        "xrefService": ["https://xref.docs.microsoft.com/query?uid={uid}"]
+    }
+}
+```
+
+## 3. Setup some basic documents
+
+We suggest starting with a basic documentation structure in the **/docs** folder. In the provided Quick Start folder we have a basic setup:
+
+```xaml
+├── docs
+│   ├── .attachments                     // All images and other attachments used by documents
+│
+│   ├── architecture-decisions
+│           └── .order
+│           └── decision-log.md       // Sample index into all ADRs
+│           └── README.md          // Landing page architecture decisions
+│
+│   ├── getting-started
+│           └── .order
+│           └── README.md          // This recipe document. Replace the content with something meaningful to the project
+│
+│   ├── guidelines
+│           └── .order
+│           └── docs-guidelines.md  // General documentation guidelines
+│           └── README.md          // Landing page guidelines
+│
+│   ├── templates                          // all templates like ADR template and such
+│           └── .order
+│           └── README.md          // Landing page templates
+│
+│   ├── working-agreements
+│           └── .order
+│           └── README.md          // Landing page working agreements
+│
+│   ├── .order                                // Providing a fixed order of files and directories
+│   ├── index.md                          // Landing page documentation
+```
+
+You can use templates like working agreements and such from the [CSE Playbook](https://github.com/microsoft/code-with-engineering-playbook/).
```

Comment:
We might want to point to docs\documentation\guidance\project-and-repositories.md, where we already summarized the templates that are scattered all over the place in the Playbook.

Reply:
added links